### ISSUE TITLE
Integrate repo with forum 

### ISSUE THREAD

**Issue by Kebap** [Mudlet/mudlet-package-repo#125]
This idea recently started floating. Not sure if it is more difficult to achieve than building anew.

It would solve registration infrastructure (forum allows multiple 3rd way authorization as well)

Also commenting on packages can be a thing in repo without having to built it from scratch.

Maybe even layout becomes a bit clearer to grasp as we align it with another Mudlet site.

**Comment by vadi2** [comment]
Interesting idea... how would it looks like in practice do you think? Threads auto-created and synced together, or?

**Comment by Kebap** [comment]
Yes either that or some even more fancy background integration, where helpful.

Need to look at forum sources, what kind of integration it offers at all.

Downside would maybe be a commitment to specific forum software.

If we want to switch at any point, it may be more difficult than before.

**Comment by vadi2** [comment]
Yeah that is true. There are also other stand-alone commenting systems that I've seen we could make use of - so if we did a lite integration with forums just for the accounts + standalone comments, that would also be a possibility. Just a thought - not suggesting this is the way.
